tailwind.config.js

```diff
@@ -8,102 +8,151 @@ module.exports = {
   theme: {
     extend: {
       colors: {
-        primary: {
-          light: '#6470F7',
-          DEFAULT: '#2F3FF4',
-          dark: '#0B1CD5',
+        'primary-blue': {
+          1: '#E2E7FF',
+          2: '#ABBAF9',
+          3: '#4659A7',
+          4: '#394B9F',
+          5: '#3D4E94',
+          6: '#29376E',
         },
-        secondary: {
-          light: '#2FEEAB',
-          DEFAULT: '#11D08D',
-          dark: '#0C9766',
+        neutral: {
+          1: '#FFFFFF',
+          2: '#BEBEBE',
+          3: '#909090',
+          4: '#5C5C5C',
+          5: '#353535',
+          6: '#000000',
+        },
+        'neutral-variant': {
+          1: '#FFFFFF',
+          2: '#BEC0C9',
+          3: '#83859B',
+          4: '#565970',
+          5: '#383849',
+          6: '#000000',
+        },
+        highlights: {
+          1: '#75B4C0',
+          2: '#6CCBB4',
+          3: '#32936F',
+          4: '#DD403A',
+          5: '#BB3F3C',
+          6: '#FFC077',
+        },
+        alternate: {
+          1: '#FDF2DF',
+          2: '#F8C67E',
+          3: '#F28F00',
+          4: '#E0F1EE',
+          5: '#59B19B',
+          6: '#2C8F75',
         },
-        light: '#F9F9FA',
-        dark: '#1F201F',
       },
       fontFamily: {
         sans: ['var(--font-sans)', 'Roboto', 'sans-serif'],
-        display: ['var(--font-display)', 'Jost', 'Roboto', 'sans-serif'],
+        display: ['var(--font-display)', 'Jost', 'Roboto', 'sans-serif', 'Montserrat'],
```

Comment:
The font Montserrat is never loaded because it's not being imported in the `_app.tsx` file. Instead of putting this here, update the font corresponding to `--font-display` in `_app.tsx`.